oshi-core/src/main/java/oshi/util/ParseUtil.java

```diff
@@ -489,4 +493,31 @@ public static int getNthIntValue(String line, int n) {
         }
         return 0;
     }
+
+    /**
+     * Removes all sub strings from the string
+     * @param original source String to remove from
+     * @param toRemove the sub string to be removed
+     * @return
+     */
+    public static String removeMatchingString(final String original, final String toRemove) {

```

Comment:
In the interest of efficiency, let's do one last change. Add a boolean argument `removeAll`.  If true, behaves as written (`String.replaceAll()`). If false, only removes the first occurrence (`String.replace()`).